code/modules/reagents/reagent_dispenser.dm

```diff
@@ -185,10 +179,9 @@
 
 
 /obj/structure/reagent_dispensers/virusfood
-	name = "Virus Food Dispenser"
-	desc = "A dispenser of virus food."
-	icon = 'icons/obj/objects.dmi'
-	icon_state = "virusfoodtank"
+	name = "virus food dispenser"
+	desc = "A dispenser of low-potency virus mutagenic."
+	icon_state = "virus_food"

```

Comment:
Whoops, forgot about that. I'll address this and your other concerns once I can.